src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/import.py

```diff
@@ -67,25 +70,33 @@ def acr_import(cmd,
                source_registry.lower() != registry.name.lower() and \
                source_registry.lower() != registry.login_server.lower():
                 raise CLIError(REGISTRY_MISMATCH)
-            source_registry = registry.id
-
-    if not is_valid_resource_id(source_registry):
-        from knack.prompting import prompt, NoTTYException
-        try:
-            source_registry = prompt(SOURCE_REGISTRY_NOT_FOUND)
-        except NoTTYException:
-            raise CLIError(NO_TTY_ERROR)
-
-    image_source = ImportSource(resource_id=source_registry, source_image=source_image)
+            source = ImportSource(resource_id=registry.id,
+                                  source_image=source_image)
+        else:
+            if source_registry_password:
+                if source_registry_username:
+                    source = ImportSource(registry_uri=source_registry_login_server,
+                                          source_image=source_image,
+                                          credentials=ImportSourceCredentials(password=source_registry_password,
+                                                                              username=source_registry_username))
+                else:
```

Comment:
@nathana1  is this true that we can specify the password without username in import?

Reply:
Yes, you need to get a token first like from 'az account get-access-token'.

Reply:
So does the CLI need to call get-access-token here and pass the token and not the passowrd. 
But its passed in the password field, correct?

Reply:
Presumes the user has jwt (that includes the username as an encoded value).  That is passed as the password.

Reply:
That aligned with how k8s was storing secrets ["myregistry.azurecr.io":{"password":"giant base64 string"}]

Reply:
What is JWT?

Reply:
JSON web token (see jwt.io for more details)